<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Leg Sleeve Control Panel</string>
  </property>
  <property name="windowIcon">
   <iconset theme="preferences-system">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="patternLabel">
        <property name="text">
         <string>Contraction Pattern</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="patternComboBox">
        <property name="toolTip">
         <string>Pattern of band contractions within a cycle.</string>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>Additive</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Independent</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="periodLabel">
        <property name="text">
         <string>Cycle Period</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="periodSpinBox">
        <property name="toolTip">
         <string>Length of a full contraction cycle in seconds.</string>
        </property>
        <property name="suffix">
         <string> s</string>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>60.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="dutyLabel">
        <property name="text">
         <string>Cycle Duty</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="dutySpinBox">
        <property name="toolTip">
         <string>Relative amount of time that the first band should contract in a cycle, as a fraction of the length of a full cycle.</string>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
        <property name="value">
         <double>0.650000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="delayLabel">
        <property name="text">
         <string>Delay per Band</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="delaySpinBox">
        <property name="toolTip">
         <string>Delay between start of contractions of successive bands, as a fraction of a full cycle.</string>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
        <property name="value">
         <double>0.200000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="uncontractedLabel">
        <property name="text">
         <string>Uncontracted Position</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="uncontractedSpinBox">
        <property name="toolTip">
         <string>Servo position when band should be fully uncontracted.</string>
        </property>
        <property name="minimum">
         <number>50</number>
        </property>
        <property name="maximum">
         <number>130</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="value">
         <number>130</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="contractedLabel">
        <property name="text">
         <string>Contracted Position</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="contractedSpinBox">
        <property name="toolTip">
         <string>Servo position when band should be fully contracted.</string>
        </property>
        <property name="minimum">
         <number>50</number>
        </property>
        <property name="maximum">
         <number>130</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
   <addaction name="actionConnect"/>
   <addaction name="separator"/>
   <addaction name="actionStartRunning"/>
   <addaction name="actionStopRunning"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit the application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset theme="audio-card">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect to the Arduino</string>
   </property>
  </action>
  <action name="actionStopRunning">
   <property name="icon">
    <iconset theme="media-playback-stop">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Stop Running</string>
   </property>
   <property name="toolTip">
    <string>Stop leg sleeve contraction</string>
   </property>
  </action>
  <action name="actionStartRunning">
   <property name="icon">
    <iconset theme="media-playback-start">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Start Running</string>
   </property>
   <property name="toolTip">
    <string>Start leg sleeve contraction</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
